src/SesWebhookCall.php

```diff
@@ -20,4 +20,10 @@ public static function storeWebhook(WebhookConfig $config, Request $request): We
             'payload' => $message->toArray(),
         ]);
     }
+
+    public function identicalMessageExists() {
```

Comment:
Add a `bool` return type.

Reply:
Will do

Reply:
return type is added, although the method name and query has changed